src/catalog/backends/common/src/main/java/org/geoserver/cloud/event/remote/resourcepool/RemoteEventResourcePoolProcessor.java

```diff
@@ -72,10 +70,33 @@ public void onCatalogRemoteModifyEvent(CatalogInfoModified event) {
                         () -> log.trace("Ignoring event from self: {}", event));
     }
 
+    @EventListener(ResetEvent.class)
+    public void onReset(ResetEvent event) {
+        log.debug("Received a ResetEvent, triggering a GeoServer reset ({})", event);
+
+        if (event.isRemote()) {
+            rawGeoServer.reset(true);
+        }
+    }
+
+    @EventListener(ReloadEvent.class)
+    public void onReload(ReloadEvent event) {
+        log.debug("Received a ReloadEvent, triggering a GeoServer reload ({})", event);
```

Comment:
same, should it go inside the `if(event.isRemote())` block

Reply:
Done in 33c46d24e75dd4a50af3d908695a1e739d75ce4b